modules/python/src2/cv2.cpp

```diff
@@ -138,6 +138,39 @@ class PyEnsureGIL
     PyGILState_STATE _state;
 };
 
+class PySafeObject
```

Comment:
Why is not `std::unique_ptr<PyObject, ...Py_CLEAR...>` in case of C++11?
Or just left a comment because their behaviors are similar.

---

BTW, It makes sense to disable copy ctor/assignment operator (C++98: move to "private:" without implementation and with comment `// = deleted`).

Reply:
True, the only difference from `std::unique_ptr` is implicit conversions to `PyObject*` and `PyObject**`, so we can safely pass this object to the Python functions directly.

Makes sense to disable copy operations. Thanks